zc_plugins/POSM/v6.0.0/admin/products_options_stock_view_all.php

```diff
@@ -532,8 +535,8 @@
 <?php
 }  // One or more products was found
 ?>
-<!-- body_eof //-->
 </div>
+<!-- body_eof //-->
 
 <!-- footer //-->
 <?php require DIR_WS_INCLUDES . 'footer.php'; ?>
```

Comment:
i would remove the end php `?>` tag.

Reply:
I'm `assuming` that you mean the one on line 542?